hub/config/src/main/java/uk/gov/ida/hub/config/ConfigModule.java

```diff
@@ -91,6 +93,20 @@ protected void configure() {
         bind(MatchingServiceAdapterService.class);
     }
 
+    @Provides
+    @Singleton
+    private Gauge getGauge() {
+        return Gauge.build("verify_config_certificate_expiry", "Timestamp of NotAfter value of X.509 certificate")
+                    .labelNames("entity_id","use","fingerprint")
+                    .register();
```

Comment:
I'd lean slightly towards creating and registering this in `PrometheusClientService` (and relying on it being a singleton) unless you really really need to inject it in tests?

Injecting a `Gauge` isn't terribly scalable because once you add a second gauge to your DI container you have to start doing funny things with `@Named`.  I see Gauge as being like Logger in this respect - you don't inject loggers, you new them up in the class that uses them.

Reply:
Done.